cinema_service/settings.py

```diff
@@ -27,7 +28,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
```

Comment:
The `DEBUG` setting is currently set to `True`. This should be set to `False` in a production environment to avoid exposing sensitive information.